community-version.py

```diff
@@ -3,6 +3,7 @@
 #code credit goes to: https://www.hackerearth.com/practice/notes/beautiful-python-a-simple-ascii-art-generator-from-images/
 #code modified to work with Python 3 by @aneagoie
 from PIL import Image
+example/make_art.py/ztm-logo.png
```

Comment:
@MehulGupta21, this line is gonna break the program. I think you meant to add a comment?

Reply:
Srry yeah you are right